### DISCUSSION TITLE
Debian 11 (Bullseye) 5.10.63+

### DISCUSSION THREAD

**Discussion by BRAVOTEAM135** [clangen/musikcube#473] [Ideas]
Hey, 

I am trying to install this on my raspberry pi. I have Debian 11 (Bullseye) 5.10.63+ and I don't see a release for this release. Is this project still active? am I missing something?

Hopefully this is a simple one. 

Thanks in advance for your help.

KUDOS

**Comment by jaadth** [comment]
Have it running on my rpi (bullseye).    Manual build works and 'musikcube/script/install-deps.sh' supports bullseye.

**Reply by BRAVOTEAM135** [reply]
How did you build I left the the make command running for 5hours and got to 15%. I'm running a RPI model b rev 2. Can I build on another computer, create the .Deb and then install on my RPI? 

Thanks for any help offered to this noob
